### DISCUSSION TITLE
Can't login anymore since updating to version 2024.6.2

### DISCUSSION THREAD

**Discussion by Vortilion** [dani-garcia/vaultwarden#4929] [Q&A]
I updated my self hosted vault warden to the newest version today (previous one was a 2023.something) because the new mobile app started showing errors after trying to connect. Some comments stated that the server also needs to be updated... So I did. But now it stopped working at all. I can access the login page, but I'm unable to login anymore. Vaultwarden runs on my QNAP NAS as a docker container. Please help!

services:
  vaultwarden:
    image: vaultwarden/server:latest
    container_name: vaultwarden
    restart: always
    environment:
      - WEBSOCKET_ENABLED=true
      - DOMAIN=https://myawesome.duckdns.org
    volumes:
      - ./vw-data:/data
    ports:
      - "50050:80"

**Comment by delta-whiplash** [comment]
Hello I have the same issue on one of my instance 
I can still use my app from my phone but if I want to log in on a phone/browser/app 
After my password prompt I have an infinity loading
